contrib/base64-cmake/README.md

```diff
@@ -0,0 +1,5 @@
+`tables.h` file was generated by the base64 library with it's makefile:
```

Comment:
Useful comment but please move it into contrib/base64-cmake-CMakeLists.txt to increase the chance that somebody reads it :-)

Reply:
Related to that, the very same file already exists in contrib/base64/lib/tables/table_dec_32bit.h. Can't we simply use it?